File: Octagon-Project/Octagon/findClassPath.vb
Public Class findClassPath
    Public Function find() As String
        Dim binFolder() As String = Split(My.Application.Info.DirectoryPath.ToString, "\")
        Dim classPath As String = ""
        For cont = 0 To binFolder.Count - 3
            If cont = 0 Then
                classPath = classPath & binFolder(cont)
            Else
                classPath = classPath & "\" & binFolder(cont)
            End If
        Next
        Return classPath
    End Function
End Class

File: OctagonC/OctagonC/FindClassPath.cs

using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.IO;
public class FindClassPath {

    private static string binFolderPath = Environment.CurrentDirectory;

    public string find(){
        String[] binFolder = binFolderPath.Split('\\');  
        string classPath = "";
        for (int cont = 0; cont <= binFolder.Length - 3; cont++)
        {
            if (cont == 0)
            {
                classPath = classPath + binFolder[cont];
            }
            else {
                classPath = classPath + Path.DirectorySeparatorChar + binFolder[cont];
            }
        }
        return classPath + Path.DirectorySeparatorChar + "Resources" + Path.DirectorySeparatorChar ;
    }
}
